backend/README.md

```diff
@@ -40,30 +42,74 @@ If not, you can install it using [rbenv](https://github.com/rbenv/rbenv):
 rbenv install 3.1.2
 ```
 
+Docker Installation Prerequisites
+
+Ensure you have Docker on your system. You can check your Docker version by running:
+
+```shell
+docker -v
+```
+If not, install docker on your system:
+
+https://www.docker.com/get-started/
+
 ### 3. Install Dependencies
 
+Ruby Installation
+
+Create a .env file in the backend folder
+```shell
+USE_DOCKER=false
+```
 Using [Bundler](https://github.com/bundler/bundler):
 
 ```shell
 bundle install
 ```
 
+Docker Installation 
+
+Create a .env file in the backend folder
+```shell
+USE_DOCKER=true
+```
```

Comment:
not needed anymore, see change in config/database.yml

Reply:
Ok